### PR TITLE
upgrade to node 6.9.1

### PR DESCRIPTION
**Pull request by sulthan-ahmed**

### CODE REVIEW

**Comment by danielepolencic** [comment]
👍 LGTM

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ node_js:
   - "4"
   - "4.4.2"
   - "5"
```
Comment (easternbloc) [inline .travis.yml]: can you remove 5 please we don't need it anymore

Comment (sulthan-ahmed) [reply .travis.yml]: cool, do you want that also in hof-middleware removed too?

Comment (easternbloc) [reply .travis.yml]: if you're pring 6 in repos just remove it as you go along 👍

Comment (sulthan-ahmed) [reply .travis.yml]: 👍

**Review by JoeChapman — COMMENTED** [review]
I think we should only be testing with two versions, and perhaps they should be 4.6.0 and 6.9.1

https://nodejs.org/en/blog/release/v4.6.0/
Of course, we'd want to updatte the docker node image for 4.6.0

**Review by JoeChapman — APPROVED** [review]

**Pull request merged**